README.md

```
@@ -20,7 +20,8 @@ Using this base unit, combinations of lengths in either [US customary units](<ht
 
 ## Minimum Supported Rust Version (MSRV)
 
-These packages have been verified to compile with **Rust 1.64** and later.
+MSRVs differ between individual packages in this workspace, but all of them are known to compile
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)